src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,70 @@
+.stopwatch {
+  position: relative;
+  width: $width;
+  height: $width;
+  border: $border-width dotted $border-color;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    height: $minutes-hand-height;
+    width: $hand-width-minutes;
+    background-color: $minutes-hand-color;
+    bottom: 50%;
+    left: calc(50% - #{$hand-width-minutes} / 2);
+    transform: translateX(-50%);
+    transform-origin: bottom center;
```

Comment:
The `transform` property is being overwritten by the second declaration on line 15. To combine translations and rotations, you should specify them in a single `transform` property, like so: `transform: translateX(-50%) rotate(0deg);`. This needs to be corrected for both the `.stopwatch__minutes-hand` and `.stopwatch__seconds-hand` elements.